sonic-slave-bookworm/Dockerfile.j2

```diff
@@ -736,3 +738,15 @@ RUN eatmydata apt install -y python-is-python3
 ARG bazelisk_url=https://github.com/bazelbuild/bazelisk/releases/latest/download/bazelisk-linux-{{ CONFIGURED_ARCH }}
 RUN curl -fsSL -o /usr/local/bin/bazel ${bazelisk_url} && chmod 755 /usr/local/bin/bazel
 {% endif -%}
+
+# Install Rust
+ARG RUST_ROOT=/usr/.cargo
+RUN RUSTUP_HOME=$RUST_ROOT CARGO_HOME=$RUST_ROOT bash -c 'curl --proto "=https" -sSf https://sh.rustup.rs | sh -s -- --default-toolchain 1.79.0 -y'
+{% if CROSS_BUILD_ENVIRON == "y" and CONFIGURED_ARCH == "armhf" %}
```

Comment:
What about CROSS_BUILD_ENVIRON = n?

Reply:
If CROSS_BUILD_ENVIRON = y, we will install toolchains for target architecture,
else, we won't.